nuitka/tools/release/Documentation.py

```diff
@@ -28,68 +28,65 @@
 from nuitka.utils.FileOperations import getFileList
 
 
+def which(prog):
+    """
+    cross version program detection.
+
+    Whenever Nuitka decides to drop Python versions older than 3.3
```

Comment:
For Nuitka this practically Python2 then, we don't support 3.2 anymore, and 3.1 and 3.0 were never considered.